index.html

```diff
@@ -36,9 +31,6 @@ <h1>Dr. Norman Borlaug</h1>
       <li>He spent most of his life saving human life</li>
   </ul>
 
-
-
-
```

Comment:
- [ ] Please try to be more professional in your comments and add solid reasons why removing the spacing or indentation is important also it seems like you are ordering, your comment should be clear, polite and professional. Kindly look at the point below, which will help you perform code reviews very well.
   - Kindly remove the spacing or indentation because it actually makes your code looks messy and makes your code base setup unprofessional.   

## Your comment

>  I suggest that you kindly remove spacing or indentations that are not necessary